lib/recipes/nursing-external/bake

```diff
@@ -31,6 +31,7 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :nursing_external) do |doc|
   book.chapters.each do |chapter|
     BakeLearningObjectives.v2(chapter: chapter, add_title: false)
     BakeNonIntroductionPages.v1(chapter: chapter)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
```

Comment:
This is an old method. We are using now more general one. It's already added:

![image](https://user-images.githubusercontent.com/40228854/225666392-9aaca0ea-499e-4e92-9d30-112548e7251e.png)

Reply:
Thanks! The old method is now removed.